vignettes/Supporting_additional_objects.Rmd

```diff
@@ -92,35 +96,45 @@ default `skimr` percentiles are returned by using `quantile()` five
 times.
 
 Next, we create a custom skimming function. To do this, we need to think about
-the many specific classes of data in the `sf` package.  The following example
-will build  support for `sfc_MULTIPOLYGON`, but note that we'll have to
-eventually think about `sfc_LINESTRING`, `sfc_POLYGON`, `sfc_MULTIPOINT` and
-others if we want to fully support `sf`.
+the many specific classes of data in the `sf` package.  From above, you can see 
+the geometry column has two classes: 1st the specific geometry type (e.g. 
+`sfc_MULTIPOLYGON` `sfc_LINESTRING`, `sfc_POLYGON`, `sfc_MULTIPOINT`) and 2nd the 
+general sfc class. Skimr will try to find a sfl() helper function for the classes 
+in the order they appear in `class(.)` (see S3 classes for more detail 
+[*Advanced R*](https://adv-r.hadley.nz/s3.html)). The following example will build  
+support for `sfc`, which encompasses all `sf` objects: `sfc_MULTIPOLYGON` 
+`sfc_LINESTRING`, `sfc_POLYGON`, `sfc_MULTIPOINT`. If we want custom skim_with 
+functions we can write sfl() helper functions for the geometry type. 
+
 
 ```{r}
 skim_sf <- skim_with(
-  sfc_MULTIPOLYGON = sfl(
+  sfc = sfl(
     n_unique = n_unique,
     valid = ~ sum(sf::st_is_valid(.)),
-    funny = funny_sf
+    crs = get_crs
   )
 )
 ```
 
 The example above creates a new *function*, and you can call that function on
-a specific column with `sfc_MULTIPOLYGON` data to get the appropriate summary 
-statistics.
+a specific column with `sfc` data to get the appropriate summary 
+statistics. The `skim_with` factory also uses the default skimrs for things 
+like factors, characters, numerics, etc. Therefore our `skim_sf` is like the regular
+`skim` function with the added ability to summarize `sfc` columns.
 
 ```{r}
-skim_sf(nc$geometry)
+skim_sf(nc)
 ```
 
+** This seems unnecessary, but maybe I'm misunderstanding something. I think the skim_with function already creates 
```

Comment:
I think you're right on it being unnecessary. Mind removing it?

@elinw  Is this ok with you if we drop this?

Reply:
This is showing specifically what the new sfl does without the distraction of the rest of the output. Yes it needs to stay for the purposes of readability and pedagogy.  Further, as noted, at least with the new (currently development) version of dplyr and other  you can't see the handling of the sf column at all and this is the only way to see this. 
```suggestion
** This seems unnecessary, but maybe I'm misunderstanding something. I think the skim_with function already creates 
```